_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,488 @@
+- The perceived visual styling of elements is the result of CSS styling applied to those elements and inherited from their [ancestor][] elements. Through manipulation of the positioning and styles of other elements, including non inherited styles of [ancestor][] elements, an element may end up with a perceived distinguishable style not resulting from its own and inherited styling. In that case, this rule might fail while the [Success Criterion 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) might still be satisfied.
+- This rule considers that [Success Criterion 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) only applies if the elements have different hues as stated in the [Understanding Success Criterion 1.4.11: Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) document. If the hues are the same and saturation or lightness are not, this creates a difference in contrast, not in the perceived color and [Success Criterion 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) does not apply. If [Success Criterion 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) applies to elements that have the same hue but different saturation and lightness, this rule will be inapplicable while [Success Criterion 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) may still fail.
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+Some examples for this rule change the default user agent styling of links. This is done to illustrate the rule. Changing the default styles of links is not endorsed by this rule. With most user agents, this rule passes with the default style; explicit styles are added to make sure the examples work as intended.
+
+- [Understanding Success Criterion 1.4.1: Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)
+- [G183: Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them](https://www.w3.org/WAI/WCAG21/Techniques/general/G183)
+- [F73: Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision](https://www.w3.org/WAI/WCAG21/Techniques/failures/F73)
+- [C15: Using CSS to change the presentation of a user interface component when it receives focus](https://www.w3.org/WAI/WCAG21/Techniques/css/C15)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This link, with a **different hue** from the other text in the same line, uses a style similar to the default styling of links which underlines them in most browsers, making it a distinguishable style.
```

Comment:
The contrast between black and blue is 8.6:1. This and many more of these examples pass just because of that. Someone can pass this example just by looking up the contrast; without having to test for underline.

Reply:
Good point. Changing the examples to use colors that differ only on their hue, and that do not meet contrast levels.